pkg/utils/common/controller.go

```diff
@@ -672,3 +674,34 @@ func (s *SuiteController) DeleteAllPipelineRunsAndTasks(namespace string) error
 
 	return nil
 }
+
+// CreateRegistryAuthSecretKeyVlaue creating key/value kubernetes secret.
```

Comment:
you have Vlaue written here instead of Value

Reply:
already fixed in a former review

Reply:
you are right, 
it was fixed only on the function name and I didn't fix in comment